### PR TITLE
fix: add back a toggle in view options

### LINKED ISSUE #738: Single Stat: Bring back Colorization options
Seems like the Colorization options disapeared from Single Stat and Graph + Single Stat as a result of this PR: https://github.com/influxdata/ui/pull/475/files#diff-4a751db90473baad8aa82064e3f662972e28ab4d8c1ca09cf97924838159aa48

We should bring it back to Single Stat and Graph + Single Stat.

image

### PR DESCRIPTION
**Pull request by drdelambre**
Closes #738 

seems like these didn't get copy pasta'd correctly

### CODE REVIEW

File: src/visualization/types/SingleStat/options.tsx [hunk 1]
```diff
@@ -46,6 +52,42 @@ const SingleStatOptions: FC<Props> = ({properties, update}) => {
     update({colors})
   }
 
+  const updateThreshold = useCallback(
+    (threshold: string) => {
+      if (threshold === THRESHOLD_TYPE_BG) {
```
Comment (genehynson) [inline src/visualization/types/SingleStat/options.tsx:57]: could this conditional... (see next comment)

File: src/visualization/types/SingleStat/options.tsx [hunk 2]
```diff
@@ -46,6 +52,42 @@ const SingleStatOptions: FC<Props> = ({properties, update}) => {
...
+        update({
+          colors: properties.colors.map(color => {
+            if (color.type !== 'scale') {
+              return {
+                ...color,
+                type: THRESHOLD_TYPE_BG,
```
Comment (genehynson) [inline src/visualization/types/SingleStat/options.tsx:63]: ...be converted to a ternary here?

I think that would eliminate a bunch of duplicate code

File: src/visualization/types/SingleStatWithLine/options.tsx [hunk 3]
```diff
@@ -128,6 +132,42 @@ const SingleStatWithLineOptions: FC<Props> = ({
     }
   }
 
+  const updateThreshold = useCallback(
+    (threshold: string) => {
+      if (threshold === THRESHOLD_TYPE_BG) {
```
Comment (genehynson) [inline src/visualization/types/SingleStatWithLine/options.tsx:137]: Same here as above

File: src/visualization/types/SingleStat/options.tsx (hunk 2 above)
Comment (drdelambre) [reply src/visualization/types/SingleStat/options.tsx:63]: totes can!

**Review by genehynson — APPROVED** [review]
🚀

**Pull request merged**